src/shared/UniversalRender.jsx

```diff
@@ -39,6 +92,8 @@ const sagaMiddleware = createSagaMiddleware(
     ...marketWatches
 );
 
+const TOP_OFFSET = 70; //this should be declared in a sass file. The way the offset 'happens' currently is via a collection of padding and margin values on different elements; pretty bad. Once that's fixed, we can move this declaration to a central location and have it all work in lockstep.
```

Comment:
I'm not seeing where this is used

Reply:
That was an accidental leave in. I decided to find < header > and use it instead.

I'll remove it. < header > *should* have a fixed height declared in a scss file, exportable to js so we can skip calculations, and constants like `TOP_OFFSET`.

Reply:
Ok, gotcha.. but depending on viewport width there may be different header heights (at least due to mobile vs. desktop views)

Reply:
> depending on viewport width

Indeed. A fixed header affects the content viewport height at all times. While I'd expect differences at breakpoints, we probably want to guard the user's content viewport height jealously. AKA, the fixed header will always only have one of a discrete set of approved by @pkattera values; all of which are specified in the `.scss` file as described above.

Hadn't figured out how we determine which one to use, but that's a problem for when we actually have them stored in the file.